### ISSUE TITLE
AJAX problem

### ISSUE THREAD

**Issue by AmazingGabriel16** [AmazingGabriel16/novelai#3]
AJAX and all the other dynamic stuff not working in API 29 and below.

Discovered issue when testing to fix issue #2 .

**Comment by AmazingGabriel16** [comment]
**Update**
_Version 1.2 (previously known as 1.0.2)_

I have done some more testing on different API levels in Android Studio and API level 29-26 are not showing the dynamic content, resulting in a plain screen that is the NAI website's background color,

However, when testing it on a real device in Android 8 (API 26), version 1.2 did indeed work properly as intended without any issues at all. Resulting in a suspicion that Android Studio is just being buggy, however of course, there are not enough test results from real world devices on a lower API level below 30 to create a educated enough conclusion to the issue.

Because of this, I am putting active code and research development on this issue on hold until more testing is completed and/or more feedback is received by the community.

Due to this, the tag CRITICAL has been removed due to the inconsistent testing results that are being observed, and sub-critical is being put in it's place instead due to those inconsistent results from testing the application, however it is still possible that it is a widespread issue. Please be sure to post feedback of the issues, such as testing results and problems in this issue thread. In the event that there is no new feedback on this issue that points to this issue being present, this issue will be closed, presumably at the end of the month (October), to allow enough users to test the version.

If you have a device that happens to use Android 10 (API 29) and below, please help by downloading version [1.2](https://github.com/AmazingGabriel16/novelai/releases/tag/1.0.2-beta) and testing out to see if the webpage inside of the application loads.

_What should a successful result look like?_
A successful result is where the page will be displayed like the normal NAI website. A unsuccessful result will mean that it is just a plain block of color that is the NAI theme.

Thank you in advance to any testers,
Gabriel

**My device test specifications**
Phone: Samsung S7 (I think, I lost track of the phones)
Android: 8.0 (API 26)
Samsung Experience: 9

**Comment by AmazingGabriel16** [comment]
Update
_Closed_

This issue is now closed as we have reached the closure date at which there have been no updates by users in response to this issue. Where if there were no updates or anything from testers, that this issue would be presumed to have just been a Android Studio emulator bug due to the issue not being present in a lower API level of Android on a physical device.

I also just noticed that the due date was not clear enough in the paragraph I had said in my previous comment and I will make sure to add solid dates or make my sentences clearer to remove any misunderstandings.

Thank you for those who tested in the event that there were any, and those who have reported the issue,
Gabriel
